CHANGELOG.md

```diff
@@ -41,7 +39,6 @@ Due to [\#5553](https://github.com/decidim/decidim/pull/5553), SSL is turned on
 - **decidim-core**: Redesign activity cards on Timeline and Activity Tabs for a consistent user experience [#5779](https://github.com/decidim/decidim/issues/5779)
 - **decidim-proposals**: Update rspec proposal_activity_cell_spec to check existence of card__content css class instead of car-data css class [#5779](https://github.com/decidim/decidim/issues/5779)
 - **decidim-comments**: Update rspec comment_activity_cell_spec to check existence of card__content css class instead of car-data css class[#5779](https://github.com/decidim/decidim/issues/5779)
-- **decidim-core**: Fix clearing the current_user after sign out [#5823](https://github.com/decidim/decidim/pull/5823)
```

Comment:
This four entries should not be removed, correct?

Reply:
Those entries are not mine and were causing conflict, and they are already in develop